### ISSUE TITLE
CI: compile and upload native-image binaries to GitHub Releases

### ISSUE THREAD

**Issue by TimoKramer** [replikativ/datahike#629]
### Describe the feature you would like to request

Like babashka or other compiled clojure projects.

### Describe the solution you would like

Just look into babashka repo and copy over stuff.

### Describe alternatives you've considered

none

**Comment by kordano** [comment]
Good idea, do you need anything from me to proceed there?

**Comment by TimoKramer** [comment]
no, I don't need anything. just wondering which target platforms we need.

- linux amd64
- linux aarch64
- macos amd64
- macos aarch64
- windows amd64
- windows aarch64
